benches/dapol.rs

```diff
@@ -1,175 +1,350 @@
-use core::fmt::Debug;
-use criterion::{criterion_group, criterion_main, BatchSize, BenchmarkId, Criterion, SamplingMode};
-use dapol::{
-    utils::get_secret, Dapol, DapolNode, RangeProofPadding, RangeProofSplitting, RangeProvable,
-    RangeVerifiable,
-};
-use digest::Digest;
-use rand::{distributions::Uniform, thread_rng, Rng};
-use smtree::{
-    index::TreeIndex,
-    traits::{ProofExtractable, Rand, Serializable, TypeName},
-};
-use std::time::Duration;
-
-// CONSTANTS
-// ================================================================================================
+mod setup;
+
+use criterion::{criterion_group, criterion_main};
+use criterion::{BenchmarkId, Criterion};
+use iai_callgrind::{black_box, library_benchmark, library_benchmark_group, main};
+
+// use std::path::PathBuf;
```

Comment:
Stray import?

Reply:
There was a reason, but I can't remember 😂 Something to do with serialization, I think. I'll remove it